R/FilterCarScore.R

```diff
@@ -14,16 +14,28 @@
 #' @template seealso_filter
 #' @export
 #' @examples
-#' task = mlr3::tsk("mtcars")
-#' filter = flt("carscore")
-#' filter$calculate(task)
-#' head(as.data.table(filter), 3)
+#' if (requireNamespace("care")) {
+#'   task = mlr3::tsk("mtcars")
+#'   filter = flt("carscore")
+#'   filter$calculate(task)
+#'   head(as.data.table(filter), 3)
+#' }
 #'
 #' ## changing filter settings
 #' filter = flt("carscore")
 #' filter$param_set$values = list("diagonal" = TRUE)
 #' filter$calculate(task)
 #' head(as.data.table(filter), 3)
+#'
+#' if (requireNamespace("mlr3pipelines") && requireNamespace("care")) {
+#'   library("mlr3pipelines")
+#'   task = mlr3::tsk("mtcars")
+#'
+#'   graph = po("filter", filter = flt("carscore"), filter.cutoff = 0.2) %>>%
```

Comment:
I'd avoid setting `filter.cutoff` as users might takes this as a "good default" then as its listed in an example.

I see you're setting a different value in each example. Should we calculate a random dynamic value and add a comment that this value should be tuned in practice?

Reply:
Adding a comment that this value should be tuned: 👍 

I don't think a random value is helping, this is just confusing.

Reply:
> I don't think a random value is helping, this is just confusing.

Agree, this might not be obvious. Maybe the combination of a hardcoded value across all examples and a comment about the tuning need might be the best middle way.